### PR TITLE
Update to version 1.0.2

### PR DESCRIPTION
**Pull request by 183amir**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,48 +1,38 @@
...
+    - {{ compiler('cxx') }}
 
 test:
   commands:
     - test -d $PREFIX/include/blitz        # [unix]
     - test -e $PREFIX/lib/libblitz.a       # [unix]
     - test -e $PREFIX/lib/libblitz.so      # [linux]
     - test -e $PREFIX/lib/libblitz.dylib   # [osx]
-    - if not exist %PREFIX%\Library\bin\blitz.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\blitz.lib exit 1  # [win]
```
Comment (4p0pt0Z) [inline recipe/meta.yaml:27]:
```suggestion
```
I don't know much about library compiling on windows, but I see you removed the parts in `bld.bat` that copied the `.dll` and `.lib` files, but kept the check for the `.lib` file here.

Comment (183amir) [reply recipe/meta.yaml:27]: I was trying to replicate these steps: https://github.com/blitzpp/blitz/blob/1.0.2/appveyor.yml but looks like cmake does not copy them.

**Comment by 183amir** [comment]
@conda-forge-admin, please rerender

**Pull request merged**